src/solana/rpc/core.py

```diff
@@ -549,11 +550,12 @@ def _simulate_transaction_body(
         accounts_encoding: str,
     ) -> Union[SimulateLegacyTransaction, SimulateVersionedTransaction]:
         commitment_to_use = _COMMITMENT_TO_SOLDERS[commitment or self._commitment]
+        accounts = None
```

Comment:
It doesn't look like `RpcSimulateTransactionAccountsConfig` will accept `None`, 
https://github.com/kevinheavey/solders/blob/46e406a405392469fa1a52f61f0b116a51c1e6a8/crates/rpc-simulate-tx-accounts-config/src/lib.rs#L27-L35

Can we default to an empty array instead?